server/sonar-server/src/main/java/org/sonar/server/rule/index/RuleIndex.java

```diff
@@ -183,8 +182,9 @@ public RuleIndex(EsClient client, System2 system2) {
     }
 
     esSearch.setQuery(boolQuery().must(qb).filter(fb));
+    System.out.println(esSearch);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Replace this use of System.out or System.err by a logger. ([squid:S106](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS106))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2Fsl%2F7.1%2FSONAR-10313%2Fremove_key_from_rule_indexes&issues=AWFL-JqXNbtQu1yJ8qeE&open=AWFL-JqXNbtQu1yJ8qeE)

[ISSUE_KEY]: AWFL-JqXNbtQu1yJ8qeE